### PR TITLE
APP-517: [C++] Different behavior of the Event demo

### PR DESCRIPTION
**Pull request by Dubland**
/cc @Slash32

### CODE REVIEW

**Comment by Dubland** [comment]
@kaa-leeroy retest this please

**Comment by Dubland** [comment]
@kaa-leeroy test this please

File: event/source/cpp/src/KaaDemo.cpp [hunk 1]
```diff
@@ -172,35 +172,33 @@ class ChatMenu {
 
 private:
     bool processCommand(const std::string &command) {
```
Comment (sparadiuk) [inline event/source/cpp/src/KaaDemo.cpp:174]: I propose to put all "if" to one construction.
Example:
bool processCommand(const std::string &command) {
        if (command.compare("1") == 0) {
            return command_join();
        } else if (command.compare("2") == 0) {
            return command_create();
        }  else if (command.compare("3") == 0) {
            return command_delete();
        } if (command.compare("4") == 0) {
            printRooms();
            return true;
        } else if (command.compare("5") == 0) {
            return false;
        } else {
            std::cout << "Unknown command " << command << '\n';
        }
        return true;
    }

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Review by rasendubi — APPROVED** [review]
Superficially looks good.

**Comment by sparadiuk** [comment]
Please check!
I saw the log in the Jira from C. It should be:
Event demo started
.....
Kaa Demo attach success
0 event listeners received

But I don't see the code in the C++ demo which out it.

I think, better will make the same log in the all languages (C++/C/Java)

**Pull request merged**